<launch>
	<node pkg="CamTrap_Viper" type="hypervisor" name="hypervisor"/>	
	<node pkg="CamTrap_Viper" type="cv" name="cv"/>
	
	<node pkg="CamTrap_Viper" type="com" name="com"/>
	<node pkg="CamTrap_Viper" type="motors_ninja" name="motors_ninja"/>
	<node pkg="CamTrap_Viper" type="log" name="log"/>
</launch>
